capstone_project/src/test/java/com/google/sps/PostTest.java

```diff
@@ -195,100 +209,4 @@ public void returnEmptyListOnInvalidSortMethod() {
 
     assertEquals(expected, actual);
   }
-
-  @Test
```

Comment:
These tests were removed because the way upvotePost works has changed, causing these tests to no longer be relevant.